### ISSUE TITLE
Player not saving field display windows

### ISSUE THREAD

**Issue by jpsluka** [CompuCell3D/CompuCell3D#395]
Windows10, CC3D 4.2.4 20210123
Player doesn't seem to be saving the layout of non-cell field windows, like windows displaying fields. Looks like any window not displaying "Cell_Field" is unsaved.

Still doesn't appear to be saving the status of the windows in a consistent manner. Might or might not at end of a run. Might or might not when stop is pressed.

**Comment by CompuCell3D** [comment]
Interesting. does it always happen or every once in a while? I tried reproducing this issue but things worked for me. Maybe you can email me simulation + steps you take to reproduce the bug?

**Comment by jpsluka** [comment]
I've sent M. a zip fie and screen shots showing the problem.

**Comment by CompuCell3D** [comment]
This happens when user-defined visualization fields are defined outside Steppable's constructor. We need a better way of handling such situation

**Comment by CompuCell3D** [comment]
So in this situation Player behaves correctly. Because users can define extra visualizations at any point in the steppable it makes sense to only restore windows for fiends that exist right after start function of the steppables runs. What do you think @jpsluka  ?

**Comment by jpsluka** [comment]
I disagree. The state of Player's windows should not depend on when or where extra visualizations are defined. If there is a window shown in player then Player should remember that. If the user defines an extra visualization and it isn't being shown in player then no reason for player to remember that field exists.
